packages/datadog_flutter_plugin/lib/src/rum/ddrum_method_channel.dart

```diff
@@ -50,116 +46,122 @@ class DdRumMethodChannel extends DdRumPlatform {
   }
 
   @override
-  Future<void> addTiming(String name) {
+  Future<void> addTiming(DateTime timeStamp, String name) {
```

Comment:
minor: should we use a consistent way of writing it? because I see  both `timeStamp` and `timestamp`

Reply:
Yeah probably....